### ISSUE TITLE
testing fresh hugo site getting error making new post

### ISSUE THREAD

**Issue by noahdesu** [halogenica/beautifulhugo#37]
[nwatkins@sapporo nohaha]$ hugo new post/test-post.md
Error: error from marshal: unsupported value type map[interface {}]interface {}: map[]

**Comment by rayjolt** [comment]
I got the same thing. To reproduce:

```
$ mkdir hugo-site
$ cd hugo-site
$ hugo new site .
$ git clone git@github.com:halogenica/beautifulhugo.git themes/beautifulhugo
$ echo "theme = \"beautifulhugo\"" >> config.toml
$ hugo new post/test-post.md
```

The problem disappeared when I changed `bigimg: [{}]` to `bigimg: []` in [archetypes/default.md](https://github.com/halogenica/beautifulhugo/blob/master/archetypes/default.md). It looks like hugo is treating the `[{}]` part as invalid. I couldn't find a way to make it work with other YAML syntax, though.

**Comment by halogenica** [comment]
The archetype is defined in yaml and expects you to use the yaml metadata format. This is why the example site has the line `metaDataFormat = "yaml"`. It looks like this doesn't play nicely with the toml format. Since this was apparently confusing I'll just remove the problematic fields from the archetype.

**Comment by halogenica** [comment]
There we go, fixed with 4946318f227d52cebc7183068016b87740974777. Hope that works for you!
